make-webpack-config.js

```diff
@@ -25,6 +26,7 @@ module.exports = function(options){
   }
 
   return {
+    devtool: isProd ? 'hidden-source-map' : 'eval',
```

Comment:
Yea, I think you got that wrong. The `hidden-source-map` has identical impact on production. Only difference is that code doesn't include `//sourceMappingUrl` so resulting file wont be automatically retrieved when devtools are open. Unfortunately there is no way in Chrome DevTools (nor other browsers) to say where to find hidden source map files making it rather useless setting.